src/faultlog/deconfig_records.cpp

```diff
@@ -98,6 +97,12 @@ void DeconfigRecords::populate(nlohmann::json& jsonNag)
                 {
                     state = stateConfigured;
                 }
+                std::stringstream ss;
```

Comment:
we could use std::format and avoid another stringstream declaration.
deconfigJson["PLID"] = std::format("0x{:x}", hwasState.deconfiguredByEid);

Reply:
value was written in decimal format I need to convert to hexa decimal.